src/process/pidfd_getfd.rs

```diff
@@ -0,0 +1,42 @@
+#![allow(unsafe_code)]
+use crate::fd::OwnedFd;
+use crate::{backend, io};
+use backend::fd::{AsFd, RawFd};
+
+/// Raw file descriptor in another process.
+pub type ForeignRawFd = RawFd;
+
+bitflags::bitflags! {
+    /// All flags are resuerved for future use.
+    pub struct PidfdGetfdFlags: backend::c::c_uint {}
+}
+
+/// `syscall(SYS_pidfd_getfd, pidfd, flags)`—Obtain a duplicate of another
+/// process's file descriptor.
+///
+/// # References
+///  - [Linux]
+///
+/// # Safety
```

Comment:
Strictly speaking, this is no longer a `Safety` comment, because the function is not `unsafe`. Could you change this to say `# Warning`, which rustix uses in a few other places for similar things?